### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by kaytcampbell**
:art:
- Addresses PR comments from https://github.com/amaabca/ama_layout/pull/135
- removed duplicate base styles
- cleaned up some syntax
- added some print styles

### CODE REVIEW

**Comment by jessedoyle** [comment]
:book:

File: app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss [hunk 1]
```diff
@@ -35,7 +35,6 @@
 
   &__button{
     @extend .button;
-    margin-bottom: 0; //assuming the button lives within a container, which causes 2 sets of margins
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss]: Does the bottom margin get zero out like the other margins now? I think the margin bottom is coming from the button tag on most button groupings currently.

**Comment by jessedoyle** [comment]
:art::art::art:

File: app/assets/stylesheets/ama_layout/_settings.scss [hunk 2]
```diff
@@ -45,7 +45,7 @@
 // 1. Global
 // ---------
 
-$global-font-size: 100%;
+$global-font-size: 14px;
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/_settings.scss]: Should we being rems here since the base font size is defined in rems?

**Comment by kathleenAMA** [comment]
:hankey: didnt really fail but I just had a few questions.

File: app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss (hunk 1 above)
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss]: the issue was that before the margin was being applied twice, so now it only gets applied once, so no override is required

File: app/assets/stylesheets/ama_layout/_settings.scss (hunk 2 above)
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/_settings.scss]: No, foundation only takes a pixel or percent value here (no rems)

**Comment by kaytcampbell** [comment]
:wave:

**Comment by kathleenAMA** [comment]
:art: :art: :art:

**Pull request merged**